RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py

```diff
@@ -12,7 +12,7 @@
 pixelTTRHBuilderWithoutAngle = copy.deepcopy(ttrhbwr)
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 from RecoTracker.TkSeedingLayers.PixelLayerTriplets_cfi import *
-from RecoPixelVertexing.PixelTrackFitting.PixelTracks_cfi import *
+from RecoPixelVertexing.PixelTrackFitting.pixelTracksDefault_cfi import *
```

Comment:
Whoops, something I missed in #17170. More proper solution would be to add `pixelTracks_cfi.py`, move
```
pixelTracks = _pixelTracksDefault.clone()
_SeedingHitSets = dict(SeedingHitSets = "pixelTracksHitQuadruplets")
trackingPhase1PU70.toModify(pixelTracks, **_SeedingHitSets)
trackingPhase2PU140.toModify(pixelTracks, **_SeedingHitSets)
```
from [RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py#L72-L76) there, and fix all imports.

@Martin-Grunewald Depending on your preference, I can do it now, after this PR, or leave it to you.